.github/workflows/migration-deploy.yaml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Migrations to Production
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    env:
+      SUPABASE_ACCESS_TOKEN: ${{ secrets.SUPABASE_ACCESS_TOKEN }}
+      SUPABASE_DB_PASSWORD: ${{ secrets.SUPABASE_DB_PASSWORD }}
+      PROJECT_ID: pfsmhuunrlzvdbteqfku
```

Comment:
Does this mean we should maybe get a Supabase instance for `develop` and for `master`? And do we want to rename `master` to `main` before we add this script?

Reply:
I wonder what is the systematic way to handle schema differences per feature branch -__-
Do you suggest the "develop" instance to be managed manually so that it kinda works with every PR in progress? Or lose the ability to have working PR previews before you merge to develop (at which moment the develop instance would have migrations applied). I personally would not overcomplicate the automation because it doesn't seem that there's a neat solution anyway